tests/test_llm_integration.py

```
@@ -198,6 +207,147 @@ async def _fake_slot(prompt: str, task_type: str, **kwargs):
+def test_model_loading_fails_gracefully(
+    mock_settings: Settings, monkeypatch: pytest.MonkeyPatch
+) -> None:
+    with patch(
+        "monGARS.core.llm_integration.AutoModelForCausalLM.from_pretrained"
+    ) as mock_load:
+        mock_load.side_effect = Exception("Model loading failed")
+        with patch("monGARS.core.llm_integration.get_settings", return_value=mock_settings):
+            llm = LLMIntegration()
+
+        class _FailingRuntime:
+            def embed(self, texts: list[str]) -> list[list[float]]:
```

Ruff (0.14.4)

Unused method argument: `texts`

(ARG002)